Zend/zend_language_scanner.l

```diff
@@ -1545,12 +1624,37 @@ NEWLINE ("\r"|"\n"|"\r\n")
 	return T_SR;
 }
 
+<ST_IN_SCRIPTING>"," {
+	if((in_class && in_const_list)) {
+		yy_push_state(ST_LOOKING_FOR_SR_NAME);
+	}
+	return ',';
+}
```

Comment:
Not sure I understand your response. In that case `in_class` and `in_const_list` should be 1, so that would push `LOOKING_FOR_SR_NAME` after the `,` - or not? How does it distinguish between `A = 1, B = 2` and `A = [1, B]` for example?